content/Self-Hosting.md

```diff
@@ -0,0 +1,85 @@
+---
+weight: 12
+title: Self Hosting
+---
+
+Follow this brief guide to download and host locally your own version of the Hyprland Wiki.  
+You can either choose a specific version, or grab the latest to date and keep it updated with just a couple of commands.
+
+## Dependencies
+
+You'll need the follwing programs:
+
+{{< tabs items="Arch,NixOS" >}}
+
+{{< tab >}}
+
+```shell
+sudo pacman -Sy git hugo awk findutils
+```
+
+{{< /tab >}}
+
+{{< tab >}}
+
+```shell
+TODO!
```

Comment:
```suggestion
nix shell nixpkgs#{gawk,go,hugo,fzf,git,findutils}
```

`awk` should be available by default along with `findutils`, but I've included them for completeness. BTW, `fzf` is not mentioned in the Arch command.

`go` is not automatically pulled into `PATH` by `hugo`, so we add it separately.